nilearn/plotting/matrix_plotting.py

```diff
@@ -455,18 +456,20 @@ def pad_contrast_matrix(contrast_def, design_matrix):
 def plot_design_matrix(
     design_matrix, rescale=True, axes=None, output_file=None
 ):
-    """Plot a design matrix provided as a :class:`pandas.DataFrame`.
+    """Plot a design matrix.
 
     Parameters
     ----------
-    design matrix : :class:`pandas.DataFrame`
+    design matrix : :class:`pandas.DataFrame` or \
+                    :obj:`str` or :obj:`pathlib.Path` to a TSV event file
```

Comment:
CSV works too right?

Reply:
We probably don't want to advertize it.

Reply:
yeah csv is fine too: adding a test to check just in case